code/modules/shuttle/cas_plane.dm

```diff
@@ -308,13 +308,11 @@
 		return
 	if(!GLOB.cameranet.checkTurfVis(get_turf_pixel(target)))
 		return
-	if(!active_weapon)
-		to_chat(source, "<span class='warning'>No active weapon selected!</span>")
-		return
```

Comment:
?

Reply:
Oh, I must have accidentally deleted that, I'll put it back in